pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn

```diff
@@ -873,10 +877,18 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
                 player, playerLevel);
 
             // If the user was temp'd, show admins who temp'd the player.
-            if (tempLevel[player] == 1 || tempLevel[player] == 2) {
-                if (Player(playerid)->isAdministrator())
+            {     
+                if (Player(playerid)->isAdministrator() && Player(playerid)->isManagement()) 
```

Comment:
This isAdministrator is redundant here. If you're management you're always an admin